filament/CMakeLists.txt

```diff
@@ -238,6 +238,7 @@ set(MATERIAL_SRCS
         src/materials/resolveDepth.mat
         src/materials/separableGaussianBlur.mat
         src/materials/skybox.mat
+        src/materials/shadowmap.mat
```

Comment:
won't we be increasing the size of the lib for a debug feature?

Reply:
yeah, a bit. I was wondering if it should be there in DEBUG only. not sure. I think it's probably small enough?

Reply:
You should check its size in the generated folders. mat files can quickly grow :/

Reply:
Yeah but this one has just one variant.